config-model/src/main/java/com/yahoo/vespa/model/container/search/ContainerSearch.java

```diff
@@ -113,6 +119,20 @@ public void getConfig(PageTemplatesConfig.Builder builder) {
         if (pageTemplates!=null) pageTemplates.getConfig(builder);
     }
 
+    private String buildGCOpts(Zone zone) {
+        Optional<String> gcopts = owningCluster.getGCOpts();
+        if (gcopts.isPresent()) {
+            return gcopts.get();
+        } else if (zone.system() == SystemName.dev) {
+            return ContainerCluster.G1GC;
+        } else if (owningCluster.isHostedVespa()) {
```

Comment:
It's okay to have this logic here since it's temporary, otherwise is should be moved config-model-fat-amended.

Reply:
Agree, hopefylly it can go away on thuesday.